### DISCUSSION TITLE
Feature: 2 or more teams in Security Onion

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#4232] [Ideas]
Hello,

By now there is no way to work in 2 or more different teams in security onion (by my current knowledge).
It was nice if I could escalate events in alerts GUI, and the case will be open in my team TheHive organization (by my mail that I connected to security onion)

Thanks :)

**Comment by cyberkimh** [comment]
Yes I'd be interested in this too please.
